### PR TITLE
Comment display + Duplicate prevention

### PR DESCRIPTION
**Pull request by shayla11**
This PR consist of the stylistic changes I made to make my comments look a lot nicer when loaded. onto the profile page. Still have some work todo, but I'm happy with them for the moment!

Secondly I implement a while loop that deleted duplicates of comments that had the potential to be displayed twice based on when/if the user submitted a new comment.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -15,6 +15,14 @@
...
+    	    float: left;
+            clear: left;
+            position: relative;
+            padding: 10px;
+            text-align: center;
+	    }
+
 		li a {
 			display: block;
 			color: black;
```
Comment (oydang) [inline portfolio/src/main/webapp/index.html:28]: Nit: formatting

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -27,18 +27,27 @@ function addRandomMovieQuote() {
...
+    while (messageList.firstChild) {
+    messageList.removeChild(commentList.firstChild);
+  }
 }
 
+//TODO: Implement a method to delete comments
+
 function createListElement(text) {
-    const liElement = document.createElement('li');
+    const liElement = document.createElement('li-comments');
```
Comment (astrelni) [inline portfolio/src/main/webapp/script.js:50]: `createElement` takes a valid HTML tag name. There is now a way to create to create custom element with something called "web components", but this is getting a bit past my level of knowledge and requires defining the custom element type as a javascript type.

I think the simpler thing to do here is create a `li` and then give it a class. Something like:

```
const liElement = document.createElement('li');
liElement.className = "user-comment";
```

Nit: I think class names in CSS are usually named in singular as opposed to plural.

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (shayla11) [reply portfolio/src/main/webapp/index.html]: On my end the index.html file shows the correct formatting. Ill re-commit and hopefully the problem is solved!

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -57,20 +65,23 @@ <h1 style="font-size:80px;">Shayla's Portfolio</h1>
...
-        <br><br><br>
-        <p>Have feedback? Write your comments here!</p>
-        <textarea name="text-input">Comment here!</textarea>
-        <br><br>
-        <input type="submit"></input>
+		<div id="center">
+			<form action="/data" method="POST">
+				<br><br><br>
+				<p>Have feedback? Write your comments here!</p>
+				<textarea name="text-input">Comment here!</textarea>
```
[the remark this answers is not in the document]
Comment (shayla11) [reply portfolio/src/main/webapp/index.html]: Thank you!

**Pull request merged**